markdown/3.x/en/ff-predictive-basket.md

```diff
@@ -0,0 +1,106 @@
+## Predictive Basket
+
+The `ff-predictive-basket` element allows for easy integration with the FACT-Finder Predictive Basket feature.
+To use it, you need a version of FACT-Finder **NG** and WebComponents `3.15.5` or higher.
+
+### Experimental Status
+
+_Note that this feature is at an experimental stage and changes may occur._
+
+Due to being experimental, the Predictive Basket feature is **not available** by default in WebComponents.
+It must be **activated** by configuring two values on **page load** as follows:
+
+```js
+document.addEventListener("ffReady", ({ factfinder }) => {
+  factfinder.communication.globalSearchParameter.version = "ng";
+  factfinder.__experimental.predictiveBasket.enable = true;
+});
+```
+
+Without this configuration the `ff-predictive-basket` element will throw errors on attempting to use it.
```

Comment:
```suggestion
Without this configuration, the `ff-predictive-basket` element will throw errors on attempting to use it.
```

Reply:
According to some style guides starting a sentence with "With" or "Without" is not very good.
The good old "If ....." seems t be the safer bet here.